### PR TITLE
TINY-8954: Adding the final link and text

### PR DESCRIPTION
**Pull request by HAFRMO**
Related Ticket: 

Description of Changes:
Add the final text and link ( Link not yet set up on the site ). No changelog needed, no tests needed.

Pre-checks:
* [x] Changelog entry added
* [x] Tests have been added (if applicable)
* [x] Branch prefixed with `feature/`, `hotfix/` or `spike/`

Review:
* [x] Milestone set
* [x] Docs ticket created (if applicable)

GitHub issues (if applicable):

### CODE REVIEW

File: modules/tinymce/src/themes/silver/main/ts/ui/promotion/Promotion.ts [hunk 1]
```diff
@@ -1,8 +1,8 @@
 import { AlloySpec, SimpleSpec } from '@ephox/alloy';
 
 // TODO final text and link to be decided in https://ephocks.atlassian.net/browse/TINY-8954
```
Comment (lostkeys) [inline modules/tinymce/src/themes/silver/main/ts/ui/promotion/Promotion.ts:3]: Maybe remove the TODO?

Comment (spocke) [reply modules/tinymce/src/themes/silver/main/ts/ui/promotion/Promotion.ts:3]: - [ ] Remove todo comment

**Review by spocke — APPROVED** [review]

**Review by TheSpyder — APPROVED** [review]

**Comment by lnewson** [comment]
This renders rather differently on different browsers because it's a font... I thought we weren't allowing that and it's why I said in the previous PR to ask about the lightning icon. For example here's how they look on different systems with fairly basic setups:

Chrome Win 10:
image

Chrome Linux:
image

Chrome macOS:
image

- [ ] Reconsider using a font based lightning icon

**Comment by Afraithe** [comment]
We went through this with product & design, they are fine with it following whatever OS/Browser will display.

**Pull request merged**